t/version-detection.t

```diff
@@ -0,0 +1,56 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+use JSON::Validator;
+
+use Mojolicious::Lite;
+get '/person'           => 'person';
+
+my $t = Test::Mojo->new;
+$t->get_ok('/person.json')->status_is(200);
+my $base_url = $t->tx->req->url->to_abs->path('/');
+my $document = $t->tx->res->json;
+
+{
+  my $jv = JSON::Validator->new;
+
+  eval {
+    $jv->load_and_validate_schema($document);
+  };
+  ok !$@, 'local schema.json' or diag $@;
+
+  is $jv->version, 7,     'detected version from local document, as draft-07';
+  is $jv->_id_key, '$id', 'detected id_key from local document, as draft-07';
+}
+
+{
+  my $jv = JSON::Validator->new(ua => $t->ua);
+
+  eval {
+    $jv->load_and_validate_schema("${base_url}person.json");
```

Comment:
You shouldn't need `$base_url`, since you pass `$t->ua` into `new()`.

Reply:
It is apparently needed; tests fail when `$base_url` is removed entirely. That seems to be the reason why the `$t->get_ok` line exists as well.